### PR TITLE
[feat] Homepage

### PR DESCRIPTION
**Pull request by Hamdrive**
## Overview

Successfully created the homepage for the mini project (Pomodoro)

Features:
1. landing page with redirect to Pomodoro app
2. Navbar with redirect to GitHub repository, and upcoming dark mode feature

## Deployment
https://drivers-pomodoro-git-homepage-hamdrive.vercel.app/

## Preview
image

### CODE REVIEW

File: src/components/Navbar/Navbar.jsx [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+          <div className={`fas fa-flag-checkered fa-1x ${styles.flag__rotate}`}></div>
+          {" "}Stint
+        </div>
+
+        <button className="hamburger">
+          <i className="fas fa-bars"></i>
+        </button>
+        <nav className="nav-list">
+          <button className="nav-list-close-btn">
+            <i className="fas fa-2x fa-times"></i>
```
Comment (junaidshaikh-js) [inline src/components/Navbar/Navbar.jsx:20]: Add title to the icons for accessibility

**Review by junaidshaikh-js — APPROVED** [review]
Great Work! 

Good use of CSS variables, clean folder structure. 

Can be merged.

**Review by bajajcodes — COMMENTED** [review]
- Code written is clean and consistency is maintained.
- Use absolute import to make importing easy.
- सुंदर work and can be merged.

**Comment by Hamdrive** [comment]
PR successfully reviewed by 2 members, and no further changes were requested.
Thus PR can be merged to dev and closed.

**Pull request merged**